### PR TITLE
Fixes #10452 - deployment status

### PR DESCRIPTION
**Pull request by tstrachota**

### CODE REVIEW

**Comment by ares** [comment]
LGTM, just OOC why adding `# TODO`s and not using factory provided by foreman-tasks?

**Comment by tstrachota** [comment]
Thanks, I couldn't access factories from foreman-tasks so I created a custom one.

**Comment by tstrachota** [comment]
rebased

File: test/unit/model/deployment_test.rb [hunk 1]
```diff
@@ -1,26 +1,35 @@
...
+      :stack => @stack
+    )
+    @valid_params = {
+      :name => 'deployment1',
+      :configuration => @config,
+      :organization => taxonomies(:organization1),
+      :location => taxonomies(:location1)
+    }
+
+    @registry = ForemanDeployments::Registry.new
```
Comment (ShimShtein) [inline test/unit/model/deployment_test.rb]: We already have a helper for this, aren't we?

File: test/unit/model/deployment_test.rb [hunk 2]
```diff
@@ -1,26 +1,35 @@
...
-        :name => 'deployment1',
-        :configuration => @config,
-        :organization => taxonomies(:organization1),
-        :location => taxonomies(:location1)
-      }
-    end
+  class FakeTask < ForemanDeployments::Tasks::BaseDefinition; end
 
+  setup do
+    @stack = ForemanDeployments::Stack.create(
```
Comment (ShimShtein) [inline test/unit/model/deployment_test.rb]: FactoryGirl?

**Comment by tstrachota** [comment]
@ShimShtein added factory girl and the helper for stubbing registries

**Comment by ShimShtein** [comment]
Merged as https://github.com/theforeman/foreman_deployments/commit/bed4193183764c61c57f83308dc72173ee6731bb

Thanks @tstrachota!

**Pull request closed without merging**